src/pixbuf.rs

```diff
@@ -237,3 +256,17 @@ impl Pixbuf {
         }
     }
 }
+
+#[test]
+fn test_new_from_vec() {
+    const NB_CHANNEL: usize = 3;
+    const WIDTH: usize = 5;
+    const HEIGHT: usize = 3;
+    let buff = [0; WIDTH * HEIGHT * NB_CHANNEL];
+    let pixbuf = Pixbuf::new_from_vec(buff.to_vec(),
+                                      false,
+                                      8,
+                                      WIDTH,
+                                      HEIGHT,
+                                      (WIDTH * NB_CHANNEL));
```

Comment:
There is at least two assert in the `new_from_vec` function. It make sure that none of them are fired calling this function this way.